C++ style comments are not allowed in ISO C90

After the commit https://github.com/RetroShare/RetroShare/commit/733f81269ccbc45c8d5e5f5807a94e685ce0c0d4 I can no longer build RS under Gentoo with the following error:
```
utility_functions.c:29:5: error: C++ style comments are not allowed in ISO C90
     // cppcheck-suppress memleak
     ^
```
Is there anything that can be done to avoid this issue?

Thanks!

--- a/supportlibs/pegmarkdown/utility_functions.c
+++ b/supportlibs/pegmarkdown/utility_functions.c
@@ -26,7 +26,7 @@ element *reverse(element *list) {
     element *new = NULL;
     element *next = NULL;
 #warning Phenom (2017-07-21): I don't know if it is a real memLeak for new. If not remove this warning and add a comment how it is deleted.
-    // cppcheck-suppress memleak
+    /* cppcheck-suppress memleak */
     while (list != NULL) {
         next = list->next;
         new = cons(list, new);

